easybuild/easyconfigs/n/nano/nano-6.4.eb

```diff
@@ -0,0 +1,28 @@
+# Contribution from IT4Innovations National Supercomputing Center, Czech Republic
+# Jakub Kropacek, 2022
+
+easyblock = 'ConfigureMake'
+
+name = 'nano'
+version = '6.4'
+
+homepage = 'https://www.nano-editor.org/'
+docurls = 'https://www.nano-editor.org/docs.php'
+description = """a simple editor, inspired by Pico"""
+
+toolchain = SYSTEM
```

Comment:
@anselmicz Is there any reason not to put this at `GCCcore`? That seems more natural to me.